### PR TITLE
Remove line from "nikic/php-parser" error message

### PR DESCRIPTION
**Pull request by mvorisek**
related discussion/issue: https://github.com/phpstan/phpstan/issues/10358#issuecomment-1873356412

### CODE REVIEW

**Comment by mvorisek** [comment]
Please advise where can I put https://phpstan.org/r/11e0c7af-bd52-4add-8de0-98397c5d492e test and on what test can I base it - I need to assert the phpstan error message.

**Comment by ondrejmirtes** [comment]
AnalyserIntegrationTest

**Comment by mvorisek** [comment]
The CI failures should be unrelated.

File: src/Parser/ParserErrorsException.php [hunk 1]
```diff
@@ -22,7 +22,7 @@ public function __construct(
 		private ?string $parsedFile,
 	)
 	{
-		parent::__construct(implode(', ', array_map(static fn (Error $error): string => $error->getMessage(), $errors)));
+		parent::__construct(implode(', ', array_map(static fn (Error $error): string => $error->getRawMessage(), $errors)));
```
Comment (ondrejmirtes) [inline src/Parser/ParserErrorsException.php:25]: I think one of the places are not tested. Please see the path where this is actually used, perhaps by totally breaking this piece of code and seeing where it fails.

File: tests/PHPStan/Analyser/AnalyserIntegrationTest.php [hunk 2]
```diff
@@ -1270,6 +1270,13 @@ public function testBug10147(): void
 		$this->assertNoErrors($errors);
 	}
 
+	public function testBug10358(): void
+	{
+		$errors = $this->runAnalyse(__DIR__ . '/data/bug-10358.php');
+		$this->assertCount(1, $errors);
+		$this->assertSame('Cannot use Ns\Foo2 as Foo because the name is already in use', $errors[0]->getMessage());
```
Comment (ondrejmirtes) [inline tests/PHPStan/Analyser/AnalyserIntegrationTest.php:1292]: Please assert the line too.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: src/Parser/ParserErrorsException.php (hunk 1 above)
Comment (mvorisek) [reply src/Parser/ParserErrorsException.php:25]: `src/Analyser/FileAnalyser.php` change is currently tested by `tests/PHPStan/Analyser/AnalyserIntegrationTest.php::testBug10358`

Comment (mvorisek) [reply src/Parser/ParserErrorsException.php:25]: The other exception is consumed in https://github.com/phpstan/phpstan-src/blob/1.10.67/src/Analyser/FileAnalyser.php#L211 but the message is not used - so there is currently not easy way to test the message, but it is tested in https://github.com/phpstan/phpstan-src/blob/1.10.67/tests/PHPStan/Analyser/AnalyserTest.php#L408.

File: tests/PHPStan/Analyser/AnalyserIntegrationTest.php (hunk 2 above)
Comment (mvorisek) [reply tests/PHPStan/Analyser/AnalyserIntegrationTest.php:1292]: done

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**